StellaNowSDKDemo/src/Messages/UserLoginMessage.cs

```diff
@@ -12,42 +12,48 @@ namespace StellaNowSDKDemo.Messages;
 
 public class UserLoginMessage : StellaNowMessageWrapper
 {
-    public UserLoginMessage(string userId, DateTime timestamp, string patronId)
+    public UserLoginMessage(string punterId, DateTime timestamp, int userGroupId, string userId)
```

Comment:
shall we keep the test project separated from SDK? changes in generated classes should not trigger release (now that we have CICD and builds from master branch). WDYT?